var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -665,12 +676,15 @@ def install(self, spec, prefix):
         # to make Boost find the user-config.jam
         env["BOOST_BUILD_PATH"] = self.stage.source_path
 
-        bootstrap = Executable("./bootstrap.sh")
-
-        bootstrap_options = ["--prefix=%s" % prefix]
+        bootstrap_options = []
```

Comment:
The `"--prefix=%s"` wasn't propagated to the `else` clause below. Was that intentional?

Reply:
(I might be saying the same thing as the above user but wanted to make sure to ask) With this change, I don't see `prefix` being specified anywhere: how is boost installing to the appropriate directory?